lib/measures/BiasedEconomizerSensorOutdoorT/resources/ControllerOutdoorAirFlow_T.rb

```diff
@@ -429,7 +405,7 @@ def econ_t_sensor_bias_ems_main_body(workspace, bias_sensor, controlleroutdoorai
                       #check if there is a compressor on the loop
                       #if there is, add code to check if the economizer should be locked out
                       main_body = main_body+"
-                        SET LOCKOUT_POS = "+name_cut(airsystem_name)+"_Ctg#{bias_sensor}_#{$faulttype}, !- <none>
+                        SET LOCKOUT_POS = "+name_cut(airsystem_name_new)+"_Ctg#{bias_sensor}_#{$faulttype}, !- <none>
```

Comment:
if the airsystem doesn't start with a number does this still work, `airsystem_name_new` will be `""`

Reply:
ahhh good catch